components/LessonsButton/LessonsButton.tsx

```diff
@@ -0,0 +1,24 @@
+import styles from "./LessonsButton.module.css";
+import { useRouter } from "next/router";
+
+export type LessonsButtonProps = {
+  labelLeft: string;
+  labelRight: string;
+  route: string;
+};
+
+function LessonsButton({ labelLeft, labelRight, route }: LessonsButtonProps) {
+  const router = useRouter();
+  return (
+    <div className={styles.container}>
+      <button className={styles.btn} onClick={() => router.push(route)}>
```

Comment:
If this button is only for navigation, use `a` or [`Link`](https://nextjs.org/docs/api-reference/next/link) instead. Links are for navigation, buttons a for general actions.

You will find good articles about this topic by searching for `anchor vs button`